dotnet/src/webdriver/Internal/Base64UrlEncoder.cs

```diff
@@ -0,0 +1,159 @@
+// Thanks to Microsoft.IdentityModel.Tokens package for this code.
```

Comment:
We need to put the Selenium Copyright at the top. I think we put the links to the methods we are drawing from in the comments to the specific methods (see below).